test/hummingbot/connector/test_markets_recorder.py

```diff
@@ -0,0 +1,220 @@
+import time
+from decimal import Decimal
+from unittest import TestCase
+from unittest.mock import patch
+
+from sqlalchemy import create_engine
+
+from hummingbot.connector.markets_recorder import MarketsRecorder
+from hummingbot.core.data_type.trade_fee import AddedToCostTradeFee
+from hummingbot.core.event.events import TradeType, OrderType, BuyOrderCreatedEvent, MarketEvent, OrderFilledEvent, \
+    BuyOrderCompletedEvent
+from hummingbot.model.order import Order
+from hummingbot.model.sql_connection_manager import SQLConnectionType, SQLConnectionManager
+from hummingbot.model.trade_fill import TradeFill
```

Comment:
Some of the imports are not sorted; `isort -m=3 -l=120 test/hummingbot/connector/test_markets_recorder.py `